packages/nimble-components/package.json

```diff
@@ -88,7 +88,7 @@
     "@types/d3-scale": "^4.0.2",
     "@types/d3-selection": "^3.0.0",
     "@types/d3-zoom": "^3.0.0",
-    "@types/markdown-it": "^13.0.8",
+    "@types/markdown-it": "^14.0.0",
```

Comment:
Don't understand why this is updated again, gonna close the PR and retrigger renovate